src/overlays/actors/ovl_Demo_Effect/z_demo_effect.c

```diff
@@ -20,121 +104,2126 @@ const ActorInit Demo_Effect_InitVars = {
     (ActorFunc)DemoEffect_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Demo_Effect/func_80970F50.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Demo_Effect/func_80970F58.s")
+// Code only matches when this is an array, but the second element isn't used. This variable assures only one jewel will
+// play SFX
+s16 sfxJewelId[] = { 0, 0 };
```

Comment:
Because data is word-aligned, you don't even need the second element. Also all of these need to be static and start with the s prefix.

Reply:
You're right on alignment and good catch on the static. I dunno why it only matches as an array, though.

Reply:
Static variables seem to just do their own thing sometimes. I can't explain it.